platforms.go

```diff
@@ -121,7 +121,7 @@ import (
 )
 
 var (
-	specifierRe    = regexp.MustCompile(`^[A-Za-z0-9_-]+$`)
+	specifierRe    = regexp.MustCompile(`^[\.A-Za-z0-9_-]+$`)
```

Comment:
This is specifying the same character differently than the line below `A-Za-z0-9_.-`